src/metabase/pulse/render/color.clj

```diff
@@ -16,16 +16,14 @@
 (def ^:private js-file-path "frontend_shared/color_selector.js")
 
 (def ^:private ^{:arglists '([])} js-engine
-  ;; The code that loads the JS engine is behind a delay so that we don't incur that cost on startup. The below
-  ;; assertion till look for the javascript file at startup and fail if it doesn't find it. This is to avoid a big
-  ;; delay in finding out that the system is broken
-  (let [file-url (io/resource js-file-path)]
-    (assert file-url (trs "Can''t find JS color selector at ''{0}''" js-file-path))
-    (let [dlay (delay
-                 (doto (js/context)
-                   (js/load-resource js-file-path)))]
-      (fn []
-        @dlay))))
+  ;; As of 2024/05/13, a single js-engine takes 3.5 MiB of memory
```

Comment:
Hm, is this a different engine from a "120mb per context" one?

Reply:
yea, the 120mb is for generating svg

Reply:
it's this one: https://github.com/metabase/metabase/blob/e2f19162be7bb5c2e2181237e5549058b3ecc0dd/src/metabase/pulse/render/js_svg.clj#L24

Reply:
I should be more specific in the comment tho. will update.

Reply:
Ahhh! It confused me since path to a file is written separately.